libs/fluxc/src/main/java/org/wordpress/android/fluxc/network/xmlrpc/comment/CommentXMLRPCClient.java

```diff
@@ -60,8 +61,8 @@ public void fetchComments(
         }
 
         params.add(site.getSelfHostedSiteId());
-        params.add(site.getUsername());
-        params.add(site.getPassword());
+        params.add(SiteModelExtensionsKt.getUserNameProcessed(site));
```

Comment:
❓  I'm curious why we don't use `site.getUserNameProcessed` here (and elsewhere)?